### ISSUE TITLE
Broken link to QAT screencast

### ISSUE THREAD

**Issue by bart0sh** [intel/intel-device-plugins-for-kubernetes#160]
We have a broken link on our demos page:
https://asciinema.org/a/H723QlyIf69H6Rpv0YihCtcKw

It leads to error page with:
```
This recording has been archived
All unclaimed recordings (the ones not linked to any user account) are automatically archived 7 days after upload.
```

**Comment by mythi** [comment]
@nolancon

**Comment by rojkov** [comment]
Fixed in #162
